tests/foreman/api/test_contentview.py

```diff
@@ -1080,6 +1080,28 @@ def test_cv_audit_scenarios(self, module_product, target_sat):
         assert not self.yumcv.read().needs_publish
 
 
+@pytest.mark.tier2
+def test_repository_rpm_size(target_sat):
```

Comment:
I would choose another test name, `test_repository_rpm_size` is a bit confusing, we don't test size of rpm repo here.
```suggestion
def test_repository_rpms_id_type(target_sat):
```